### PR TITLE
chore(cart): Clean up data models

### PR DESCRIPTION
**Pull request by olivermrbl**
Clean up data models in Cart Module according to latest established conventions

### CODE REVIEW

File: packages/cart/src/models/address.ts [hunk 1]
```diff
@@ -20,37 +19,37 @@ export default class Address {
...
   @Property({ columnType: "text", nullable: true })
-  postal_code?: string | null
+  postal_code: string | null = null
 
   @Property({ columnType: "text", nullable: true })
-  phone?: string | null
+  phone: string | null = null
 
   @Property({ columnType: "jsonb", nullable: true })
   metadata?: Record<string, unknown> | null
```
Comment (riqwan) [inline packages/cart/src/models/address.ts]:
```suggestion
  metadata: Record<string, unknown> | null = null
```

missed a spot

File: packages/cart/src/models/shipping-method.ts [hunk 2]
```diff
@@ -24,18 +24,18 @@ export default class ShippingMethod {
   @Property({ columnType: "text" })
   cart_id: string
 
-  @ManyToOne(() => Cart, {
-    onDelete: "cascade",
+  @ManyToOne({
+    entity: () => Cart,
     index: "IDX_shipping_method_cart_id",
-    nullable: true,
+    cascade: [Cascade.REMOVE, Cascade.PERSIST],
```
Comment (riqwan) [inline packages/cart/src/models/shipping-method.ts:30]: q: shouldn't we be cascading soft deletes on relations?

**Review by riqwan — APPROVED** [review]
lgtm!

File: packages/cart/src/models/shipping-method.ts (hunk 2 above)
Comment (olivermrbl) [reply packages/cart/src/models/shipping-method.ts:30]: Yes, I will need to make the models soft-deletable, but currently awaiting Adrien's, that will create all related methods under the hood.

**Pull request merged**